.vib/contour/vib-publish.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/contour"
         },
-        "runtime_parameters": "ImNvbnRvdXIiOgogICJzZXJ2aWNlIjoKICAgICJwb3J0cyI6CiAgICAgICJ4ZHMiOiA4MAogICAgInR5cGUiOiAiTG9hZEJhbGFuY2VyIgoiZW52b3kiOgogICJzZXJ2aWNlIjoKICAgICJ0eXBlIjogIk5vZGVQb3J0Ig==",
+        "runtime_parameters": "Y29udG91cjoKICBzZXJ2aWNlQWNjb3VudDoKICAgIGNyZWF0ZTogdHJ1ZQogICAgYXV0b21vdW50U2VydmljZUFjY291bnRUb2tlbjogdHJ1ZQogIHBvZFNlY3VyaXR5Q29udGV4dDoKICAgIGVuYWJsZWQ6IHRydWUKICAgIGZzR3JvdXA6IDEwMDIKICBjb250YWluZXJTZWN1cml0eUNvbnRleHQ6CiAgICBlbmFibGVkOiB0cnVlCiAgICBydW5Bc1VzZXI6IDEwMDIKICBzZXJ2aWNlOgogICAgcG9ydHM6CiAgICAgIHhkczogODAKICAgIHR5cGU6IExvYWRCYWxhbmNlcgplbnZveToKICBlbmFibGVkOiB0cnVlCiAgc2VydmljZToKICAgIHR5cGU6IENsdXN0ZXJJUAogICAgZXh0ZXJuYWxUcmFmZmljUG9saWN5OiAiIgogICAgcG9ydHM6CiAgICAgIGh0dHA6IDgwCmV4dHJhRGVwbG95OgotIGFwaVZlcnNpb246IGFwcHMvdjEKICBraW5kOiBEZXBsb3ltZW50CiAgbWV0YWRhdGE6CiAgICBsYWJlbHM6CiAgICAgIGFwcDoga3VhcmQKICAgIG5hbWU6IGt1YXJkCiAgc3BlYzoKICAgIHJlcGxpY2FzOiAzCiAgICBzZWxlY3RvcjoKICAgICAgbWF0Y2hMYWJlbHM6CiAgICAgICAgYXBwOiBrdWFyZAogICAgdGVtcGxhdGU6CiAgICAgIG1ldGFkYXRhOgogICAgICAgIGxhYmVsczoKICAgICAgICAgIGFwcDoga3VhcmQKICAgICAgc3BlYzoKICAgICAgICBjb250YWluZXJzOgogICAgICAgIC0gaW1hZ2U6IGdjci5pby9rdWFyLWRlbW8va3VhcmQtYW1kNjQ6MQogICAgICAgICAgbmFtZToga3VhcmQKLSBhcGlWZXJzaW9uOiB2MQogIGtpbmQ6IFNlcnZpY2UKICBtZXRhZGF0YToKICAgIGxhYmVsczoKICAgICAgYXBwOiBrdWFyZAogICAgbmFtZToga3VhcmQKICBzcGVjOgogICAgcG9ydHM6CiAgICAtIHBvcnQ6IDgwCiAgICAgIHByb3RvY29sOiBUQ1AKICAgICAgdGFyZ2V0UG9ydDogODA4MAogICAgc2VsZWN0b3I6CiAgICAgIGFwcDoga3VhcmQKICAgIHNlc3Npb25BZmZpbml0eTogTm9uZQogICAgdHlwZTogQ2x1c3RlcklQCi0gYXBpVmVyc2lvbjogbmV0d29ya2luZy5rOHMuaW8vdjEKICBraW5kOiBJbmdyZXNzCiAgbWV0YWRhdGE6CiAgICBuYW1lOiBjb250b3VyLXZpYi10ZXN0CiAgICBsYWJlbHM6CiAgICAgIGFwcDoga3VhcmQKICBzcGVjOgogICAgZGVmYXVsdEJhY2tlbmQ6CiAgICAgIHNlcnZpY2U6CiAgICAgICAgbmFtZToga3VhcmQKICAgICAgICBwb3J0OgogICAgICAgICAgbnVtYmVyOiA4MA==",
```

Comment:
I think you forgot to add the following entries:
```diff
contour:
+ enabled: true
+ manageCRDs: true
+ rbac:
+  create: true
```

Additionally, you can safely delete the entry for `envoy.service.externalTrafficPolicy`. The comment states it does only apply for LB or NodePorts:

https://github.com/bitnami/charts/blob/main/bitnami/contour/values.yaml#L749

Reply:
You are right, thanks for the catch!

Reply:
After testing `envoy.service.externalTrafficPolicy`, it seems the chart uses it regardless of the service type. Reverter the change for that param.